package/build-wheels-linux.sh

```diff
@@ -0,0 +1,229 @@
+mkdir build
+cd build
+cmake ..
+
+ls -ltrh $CMAKE_PREFIX_PATH
+
+cd $CURRDIR
+# ----------- Install CERES solver -------------------------------------------------------
+yum install libeigen3-dev # was not in COLMAP instructions
+yum install libatlas-base-dev libsuitesparse-dev
+yum install libgoogle-glog-dev libgflags-dev # was not in COLMAP instructions
+git clone https://ceres-solver.googlesource.com/ceres-solver
+cd ceres-solver
+#git checkout $(git describe --tags) # Checkout the latest release
+# the tip of master is currently bugged
+git checkout c2fab6502e5a341ff644c2bb2c5171ebd882b2d6
+mkdir build
+cd build
+cmake .. -DBUILD_TESTING=OFF \
+         -DBUILD_EXAMPLES=OFF \
+         -DEigen3_DIR=$CMAKE_PREFIX_PATH
+make -j$(nproc)
+make install
+
+
+cd $CURRDIR
+
+echo ""
+echo "PYTHON_EXECUTABLE:${PYTHON_EXECUTABLE}"
+echo "PYTHON_INCLUDE_DIR:${PYTHON_INCLUDE_DIR}"
+echo "PYTHON_LIBRARY:${PYTHON_LIBRARY}"
+echo ""
+
+# ---------- Fix broken dependencies -----
+
+# try new boost install
+yum install libboost-all-dev
+yum install git
+yum install cmake
+yum install build-essential
+yum install libboost-program-options-dev
+yum install libboost-filesystem-dev
+yum install libboost-graph-dev
+yum install libboost-system-dev
+yum install libboost-test-dev
+yum install libeigen3-dev
+yum install libsuitesparse-dev
+# yum install libfreeimage-dev
+yum install libgoogle-glog-dev
+yum install libgflags-dev
+yum install libglew-dev
+yum install qtbase5-dev
+yum install libqt5opengl5-dev
+yum install libcgal-dev
+yum install libcgal-qt5-dev
+
+yum -y install freeimage
+
+### ------ Build FreeImage from source and install --------------------
+cd $CURRDIR
+wget http://downloads.sourceforge.net/freeimage/FreeImage3180.zip
+unzip FreeImage3180.zip
+cd FreeImage
+make
+make install
+
+# Install GLEW
+yum -y install glew-devel
+
+# ---------------- Clone COLMAP ----------------------------------------------------------
+cd $CURRDIR
+git clone https://github.com/colmap/colmap.git
+cd colmap
+git checkout dev
+
+# ----------- Build COLMAP ------------------------------------------------------------
+mkdir build/
+cd build/
+cmake .. -DCMAKE_BUILD_TYPE=Release \
+         -DBoost_USE_STATIC_LIBS=ON \
+         -DBOOST_ROOT=/usr/local \
+         -DEIGEN3_INCLUDE_DIRS=/eigen-3.3.9
+
+if [ $ec -ne 0 ]; then
+    echo "Error:"
+    cat ./CMakeCache.txt
+    exit $ec
+fi
+set -e -x
+
+make -j$(nproc) install
+
+mkdir -p /io/wheelhouse
+
+# ----------- Build pycolmap wheel -----------------------------------------------------
+cd /io/
+#cp package/setup_centos.py setup.py
+cat setup.py
+
+PLAT=manylinux2014_x86_64
+#"${PYBIN}/python" setup.py bdist_wheel --python-tag=$PYTHONVER --plat-name=$PLAT
+EIGEN3_INCLUDE_DIRS="/eigen-3.3.9" "${PYBIN}/python" setup.py bdist_wheel --plat-name=$PLAT #--python-tag=$PYTHONVER
+
+# Bundle external shared libraries into the wheels
+for whl in ./dist/*.whl; do
+    auditwheel repair "$whl" -w /io/wheelhouse/ --no-update-tags
```

Comment:
Nice -- curious what `--no-update-tags` does?

Reply:
Without `--no-update-tags`, auditwheel updates the wheel filename by appending alias tags (e.g. `manylinux_2_17_x86_64` for `manylinux2014_x86_64`) so we end up for example with `pycolmap-0.0.2-cp37-cp37m-manylinux2014_x86_64.manylinux_2_17_x86_64.whl`. This is confusing and can break pip/pypi checks.

Reply:
Got it -- thanks!